NetcoinCLI/Program.cs

```diff
@@ -6,6 +6,114 @@ class Program
     {
         static void Main(string[] args)
         {
+            //TODO Fix a Logo for bank in ASCII.           
+            var isRunning = true;
+            Console.WriteLine($"\tLäser in bankdata.txt..." +
+                                                   $"\n\t Antal kunder: {1} " +
+                                                   $"\n\t Antal konton: {2} " +
+                                                   $"\n\t Totalt saldo: {3}");
+            while (isRunning)
+            {
+                try
+                {
+                    //TODO Have variables to fill out how many customers, accounts and total sum of all accounts money.
```

Comment:
Break out into a new menu class, with separate methods. Example, class Menu, void GetConsoleInput(), void HandleMenuSelection() or some other name that makes sense.